package.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Kindly personalize your readme file to your project🙏. The readme file is an essential part of each repository as it helps document what the repository is all about to visitors. Currently, your readme file has yet to be touched. Kindly make sure to go through each section and follow the instructions provided in the comments. When you submit your project for review, You should have correctly filled in each section and removed all comments👍